torch/csrc/api/src/nn/modules/normalization.cpp

```diff
@@ -0,0 +1,45 @@
+#include <torch/nn/modules/normalization.h>
+
+#include <torch/cuda.h>
+#include <torch/utils.h>
+#include <torch/nn/init.h>
+
+#include <ostream>
+#include <utility>
+
+namespace F = torch::nn::functional;
+
+namespace torch {
+namespace nn {
+
+LayerNormImpl::LayerNormImpl(const LayerNormOptions& options_) : options(options_) {
```

Comment:
based on clang-tidy:
```
{
    path: 'torch/csrc/api/src/nn/modules/normalization.cpp',
    start_line: 15,
    end_line: 15,
    start_column: 30,
    end_column: 30,
    annotation_level: 'failure',
    message: '[[modernize-pass-by-value]] warning: pass by value and use std::move'
  },
```
we should probably add ` // NOLINT(modernize-pass-by-value)` at the end of this line.